Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/components/CaseDetailsController.java

```diff
@@ -159,7 +161,8 @@ public void onEvent(Event event) throws Exception {
 						updateActivityToAttributeMap(traceID, (TraceBPMNDiagram)result.getAbstraction().getDiagram());
 						parent.showTrace(result.getVisualizedText());
 					} catch (Exception e) {
-						Messagebox.show(e.getMessage());
+						// LOGGER.error(e.getMessage());
```

Comment:
Can this be removed, or better, uncommented? If uncommented, include `e` in the logging call.